src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+  const discountik = 20;
+  const discountishche = 50;
```

Comment:
not the best variable names, but i get the joke

Reply:
> not the best variable names, but i get the joke

It is just my evening mood - I promise to use better ones next time :)